draft-ietf-tls-tls13.md

```diff
@@ -5924,6 +5924,21 @@ server-to-client encryption keys because that would entail
 the reuse of those keys. This parallels the use of the early
 application traffic keys only in the client-to-server direction.
 
+## Static (EC)DH Shares
+
+For various reasons, some implementations may be tempted to use static
+(EC)DH private keys. Using such keys lowers the security guarantees of
+TLS 1.3. Adversaries that get access to the static (EC)DH private key
+can get access to the content of the communication. Adversaries that
+acquire the key in real-time can compromise the confidentiality of the
+conversation. Adversaries that acquire the key later can use it to
+access the content of recorded sessions, thus breaking the forward
+secrecy promise of the protocol. TLS 1.3 clients should detect the use
+of static (EC)DH keys by corresponding servers, and should treat servers
+using such keys as compromised. Clients can perform this detection by
```

Comment:
"treat as compromised" is somewhat vague; it might be better to also add that clients should assume that confidentiality is not protected on connections to such servers, etc.